nano/core_test/node.cpp

```diff
@@ -1862,6 +1866,12 @@ TEST (node, rep_remove)
 	ASSERT_EQ (keys_rep1.pub, reps[0].account);
 	ASSERT_EQ (*channel_rep1, reps[0].channel_ref ());
 
+	// Modify last_packet_received so the channel is removed faster
```

Comment:
What is the effect of this block of code? Is it required?

Reply:
Which block of code are you referring to? The set_last_packet_received part? That was originally in this test when it was still using UDP. My guess is, that setting the set_last_packet_received will cause the removal of the channel_tcp because of inactivitiy. If the channel is removed, then the rep_crawler will remove the rep too.